nx/test/nx/defn/evaluator_test.exs

```diff
@@ -4,6 +4,8 @@ defmodule Nx.Defn.EvaluatorTest do
   alias Nx.Tensor, as: T
   import Nx.Defn
 
+  defp s(str), do: String.replace(str, "\r\n", "\n")
```

Comment:
I don't think this fix is easy to maintain. If we're going to support testing on Windows we'll also need a GitHub Actions workflow which runs them. Otherwise, each new test is probably not having this function being called anyway because it doesn't breaking on Mac or Linux.

@josevalim is there another fix around this? Perhaps an ExUnit.CaptureIO config or something?

Reply:
I think we need it for `capture_io` but it should not be necessary for inspect.